### DISCUSSION TITLE
Rails ERB files: How to get basic HTML editor behavior

### DISCUSSION THREAD

**Discussion by markoa** [zed-industries/zed#6655] [Feature Requests]
Let's say I'm in a Rails project and open a `*.html.erb` file. Zed correctly detects the ERB file type. However if I type `<p>` and press return for example, the editor does not indent the next line. Whose responsibility is this, Zed's or language server's? Looking for the right place to tweak settings.

**Comment by maxbrunsfeld** [comment]
This is a Zed bug. It seems like it applies to plain HTML files too. Our parsing of tags that are unclosed at the top level seems incorrect.  I think we'll need to fix this in the HTML parser we use, [`tree-sitter-html`](https://github.com/tree-sitter/tree-sitter-html).

**Reply by markoa** [reply]
thanks @maxbrunsfeld I'll be waiting for that fix then

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
